src/main/java/org/opensearch/performanceanalyzer/http_action/config/PerformanceAnalyzerConfigAction.java

```diff
@@ -63,24 +64,63 @@
             "batchMetricsRetentionPeriodMinutes";
     public static final String PERFORMANCE_ANALYZER_CONFIG_ACTION =
             "PerformanceAnalyzer_Config_Action";
-    public static final String RCA_CONFIG_PATH = "/_opendistro/_performanceanalyzer/rca/config";
-    public static final String PA_CONFIG_PATH = "/_opendistro/_performanceanalyzer/config";
-    public static final String LOGGING_CONFIG_PATH =
-            "/_opendistro/_performanceanalyzer/logging/config";
-    public static final String BATCH_METRICS_CONFIG_PATH =
-            "/_opendistro/_performanceanalyzer/batch/config";
-
-    private static final List<Route> ROUTES =
+
+    public static final String RCA_CONFIG_PATH = RestConfig.PA_BASE_URI + "/rca/config";
+    public static final String PA_CONFIG_PATH = RestConfig.PA_BASE_URI + "/config";
+    public static final String LOGGING_CONFIG_PATH = RestConfig.PA_BASE_URI + "/logging/config";
+    public static final String BATCH_METRICS_CONFIG_PATH = RestConfig.PA_BASE_URI + "/batch/config";
+
+    public static final String LEGACY_RCA_CONFIG_PATH =
```

Comment:
These `LEGACY_PA_BASE_URI` should really be `LEGACY_OPENDISTRO_PA_BASE_URI`. We weren't super clear in the doc, I fixed it in https://github.com/opensearch-project/opensearch-plugins/pull/33.